### PR TITLE
Issue120

### PR DESCRIPTION
**Pull request by FlyingBird95**
Fixed issue 120.

Issue was caused by the fact that the data of all checkboxes was send as you click only one checkbox.
This has now changed to send only the data from that checkbox.

### CODE REVIEW

**Comment by mircealungu** [comment]
You beat me with a few minutes *@kloostert! *
i was in the middle of looking at the PR.
Also, fast reaction time with the bug fixing @FlyingBird95 !

File: flask_monitoringdashboard/templates/fmd_rules.html [hunk 1]
```diff
@@ -43,17 +43,16 @@
 
 {% block script %}
```
[the remark this answers is not in the document]
Comment (FlyingBird95) [reply flask_monitoringdashboard/templates/fmd_rules.html]: the block script is defined in fmd_base. it ensures that first the required packages (i. e. jquery) are loaded before the custom jquery is executed

**Comment by kloostert** [comment]
Ah, I hope I did not prematurely accept the PR then. 
Next time I'll wait a bit ;)

**Comment by mircealungu** [comment]
no. no. i didn't find anything :)
don't wait for the slow ones like me!

File: flask_monitoringdashboard/database/median_group.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+def get_median_group(db_session, *where):
+    """ Return the median for a specific endpoint within a certain time interval.
+    If date_from is not specified, all results are counted
+    :param db_session: session for the database
+    :param where: additional where clause
+    """
+    result = db_session.query(FunctionCall.endpoint, FunctionCall.execution_time). \
+        filter(*where).order_by(FunctionCall.endpoint).all()
+    # result is now a list of tuples per request.
+    combine = {}
```
Comment (oriontvv) [inline flask_monitoringdashboard/database/median_group.py]: Sorry, I just want to notice that you can use defaultdict pattern here. It is more efficient (without multiple lookup via *in*) and readable
```
from collections import defaultdict
combine = defaultdict(list)
for key, value in result:
    combine[key].append(value)
```

**Pull request merged**